docker-compose.yml

```diff
@@ -1,20 +1,5 @@
 version: "3"
 services:
-  web:
```

Comment:
This has to be put back before merging - it was easier for me to work with dockerized CockroachDB and Phoenix app outside container.